### PR TITLE
Update description of the config `server.cypher.parallel.worker_limit`

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1104.surge.sh

**Review by renetapopova — APPROVED** [review]
Thanks, @NataliaIvakina. This needs to be updated in the neo4j repo as well.

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 1]
```diff
@@ -1610,7 +1610,8 @@ a| Number of threads to allocate to Cypher worker threads for the parallel runti
 If set to a positive number, that number of workers will be started.
 If set to `0`, one worker will be started for every logical processor available to the Java Virtual Machine.
 
-If set to a negative number we will subtract the value from the number of logical processors available; for example, say Neo4j is running on a server with 16 available processors, using `server.cypher.parallel.worker_limit = -1` would then mean that 15 threads are available for the parallel runtime.
+If set to a negative number, that number will be subtracted from the total number of logical processors available on the server.
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:1613]:
```suggestion
If set to a negative number, the total number of logical processors available on the server will be reduced by the absolute value of that number.
```

**Review by renetapopova — APPROVED** [review]
We can update it as in the code and merge it.

**Comment by NataliaIvakina** [comment]
We've updated the description in the source code and merged into `dev` branch, and that means the update will be visible in the new release 5.14, not now. Is that correct?

**Comment by renetapopova** [comment]
> We've updated the description in the source code and merged into `dev` branch, and this means the update will be visible in the new release 5.14, not now. Is this correct?

We can cherry-pick it into 5.x, as it is not correct. The source code is not visible.

**Pull request merged**